src/views/seeds/View.svelte

```diff
@@ -99,7 +101,9 @@
     <div class="fields">
       <!-- Seed Address -->
       <div class="txt-highlight">Address</div>
-      <SeedAddress {seed} port={seed.node.port} />
+      <SeedAddress
+        {seed}
+        port={seed.node.port ?? config.seeds.defaultHttpdPort} />
       <!-- Seed ID -->
       <div class="txt-highlight">Seed ID</div>
       <div class="seed-label">
```

Comment:
We should change `formatSeedId` to `formatNodeId` here

Reply:
55b07766a9d5ba3a93f0f4a1e31db8355f0cd0ba